weather.py

```diff
@@ -57,7 +63,17 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    weather_data = []
+    with open(csv_file, encoding="utf-8") as csv_file:
+        reader = csv.reader(csv_file)
+        header = next(reader)
+        for line in reader:
+            if not line:
+                continue
+            line[1] = int(line[1])
+            line[2] = int(line[2])
+            weather_data.append(line)
```

Comment:
To improve code readability, try to store updated values in a new variable, instead of updating the original variable, e.g.
```suggestion
        for line in reader:
            if not line:
                continue
            new_line[1] = int(line[1])
            new_line[2] = int(line[2])
            weather_data.append(new_line)
```